src/main/resources/stubs/bytes/bytes.gobra

```diff
@@ -0,0 +1,27 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in https://golang.org/LICENSE
+
+// Signatures for the public declarations in file
+// https://github.com/golang/go/blob/master/src/bytes/bytes.go
+
+// Package bytes implements functions for the manipulation of byte slices.
+// It is analogous to the facilities of the strings package.
+package bytes
+
+import (
+	// "internal/bytealg"
+	// "unicode"
+	// "unicode/utf8"
+)
+
+// Equal reports whether a and b
+// are the same length and contain the same bytes.
+// A nil argument is equivalent to an empty slice.
+requires forall i int :: 0 <= i && i < len(a) ==> acc(&a[i], _)
+requires forall i int :: 0 <= i && i < len(b) ==> acc(&b[i], _)
+decreases _
```

Comment:
Why do we assume termination here instead of proving it?

Reply:
I tried to prove it first, but it cannot be deduced

Reply:
I can show you the problem in 20 minutes, (I need to wait for gobra to terminate)

Reply:
![image](https://user-images.githubusercontent.com/36457952/147940765-8de95507-2b97-459f-949e-71b79aa5258e.png)
![image](https://user-images.githubusercontent.com/36457952/147940804-b65da78e-2b75-46a5-bacd-23517426b5bd.png)

Reply:
that's the issue I meant

Reply:
I see. The cause for this is that the conversions to string in `string(a)` and `string(b)` are translated to a function call at the Viper level. That function is defined in the function `byteSliceToStrFuncGenerator` in the class `StringEncoding`. Currently, the encoding does not have a termination measure, but we can assume that it always terminates. As such, adding a wildcard termination measure to that function should fix this issue.

Reply:
ok, in that case we can leave it as it is!

Reply:
or do you mean adding the termination measure to the StringEncoding function?

Reply:
I would add the termination measure to the StringEncoding to avoid similar problems in the future

Reply:
Done